x-pack/plugins/task_manager/server/lib/log_health_metrics.test.ts

```diff
@@ -39,16 +39,16 @@ describe('logHealthMetrics', () => {
 
     // We must change from OK to Warning
     (calculateHealthStatus as jest.Mock<HealthStatus>).mockImplementation(() => HealthStatus.OK);
-    logHealthMetrics(health, logger, config);
+    logHealthMetrics(health, logger, config, true);
```

Comment:
Maybe not with this PR but i think we should pass these arguments as an object (monadic).